### ISSUE TITLE
scrape_interval configuration to be updated while creating ADOT add on in EKS

### ISSUE THREAD

**Issue by vbhatt91** [aws-observability/aws-otel-community#640]
**Is your feature request related to a problem? Please describe.**
I would like to modify or keep custom value for 'scrape_interval' parameter in adot collector through the configuration from the add-on.

**Describe the solution you'd like.** 
To have this configuration editable under optional configuration settings (configuration values) while creating the ADOT add on in EKS cluster.  

**Describe alternatives you've considered.**
Manually editing the 'opentelemetrycollector' k8s object to update the values using below command. 
--> $ kubectl edit opentelemetrycollector/my-collector-amp -n  opentelemetry-operator-system

**Comment by mhausenblas** [comment]
Do you mean you want to express this in the [advanced config](https://aws-otel.github.io/docs/getting-started/adot-eks-add-on/add-on-configuration)?

**Comment by vbhatt91** [comment]
> Do you mean you want to express this in the [advanced config](https://aws-otel.github.io/docs/getting-started/adot-eks-add-on/add-on-configuration)?

Yes, if it is available via directly in [advanced config](https://aws-otel.github.io/docs/getting-started/adot-eks-add-on/add-on-configuration) then we can manage the parameter from Add on configuration itself.

**Comment by teokeecheng** [comment]
Currently the scrape interval is 15s, and it is too frequent and it cost a huge amount of money due to the usage of Amazon Managed Prometheus. Allowing us to update at least this scrape_interval allows more flexibility.
